packages/rke2-canal/charts/templates/daemonset.yaml

```diff
@@ -41,6 +41,12 @@ spec:
           operator: Exists
         - effect: NoExecute
           operator: Exists
+        - key: "node-role.kubernetes.io/control-plane"
```

Comment:
This appears to already tolerate all NoExecute/NoSchedule taints, regardless of key?